api/src/main/proto/optimizer.proto

```diff
@@ -76,16 +76,33 @@ service Optimizer {
 
     // commands/queries to OASIS
     rpc startOptimization (StartOptimizationCommandDTO) returns (StartOptimizationResponseDTO) {}
-    rpc StopOptimization (StopOptimizationCommandDTO) returns (StopOptimizationResponseDTO) {}
+    rpc stopOptimization (StopOptimizationCommandDTO) returns (StopOptimizationResponseDTO) {}
 
     // responses to OASIS supplied queries
     rpc offerSimulationResult (SimulationResponseDTO) returns (SimulationResultConfirmDTO) {}
     rpc offerSimulationConfig (NodeStatusCommandOrResponseDTO) returns (NodeChangeConfirmDTO) {}
     rpc offerErrorResult (ErrorResponseDTO) returns (ErrorConfirmDTO) {}
 
-    rpc sendMessage (MessageCommandDTO) returns (MessageReponseDTO) {}
+    rpc sendMessage (MessageCommandDTO) returns (MessageResponseDTO) {}
     rpc updateNode (NodeStatusCommandOrResponseDTO) returns (NodeChangeConfirmDTO) {}
+    rpc autoConfigure (NodeStatusCommandOrResponseDTO) returns (NodeChangeConfirmDTO) {}
```

Comment:
You are right, I will add now response. Node change != configuration change

Reply:
However, in current reference, the two model are more correlated when we do operation on that. I will propose a newer model as I got time to think about it more. I want the configuration only exist on proxy node. it will be: The two model(connection metadata, problem/plugin setup) stages will always exist the only difference is when we doing execution, the configuration is on our execution table. In that case, registration is just put that configuration into the table.